rancher/ecm-distro-tools
# Command to check Rancher dependencies for a RC

#### Proposed Changes ####

- Export a md formatted file and check if repository by given commit contains `dev-` and `rc` tags in a sort of files
- Added command line file
- Added logic to validate all given files 

#### Types of Changes ####

- Command line 

#### Testing ####
How to test:

1 - Compile ecm distro-tools with this branch `make`
2 - Repo with wrong deps: 
Local
```
./cmd/rancher_release/bin/rancher_release-linux-amd64 check-rancher-rc-deps 
```
Remote
```
./cmd/rancher_release/bin/rancher_release-linux-amd64 check-rancher-rc-deps -c aa6b3fdb5b402b7ffbf0b7dfd506e9571c8759e5 
```
Example of exportation:

#### Linked Issues ####

https://github.com/rancher/ecm-distro-tools/issues/272

#### User-Facing Change ####

#### Further Comments ####
